src/dags/dag.py

```diff
@@ -614,15 +619,60 @@ def format_list_linewise(seq: Sequence[object]) -> str:
     ).format(formatted_list=formatted_list)
 
 
-def get_input_types(func: GenericCallable) -> dict[str, type]:
-    """Get the input types annotations of a function.
+def _verify_annotations_are_strings(
```

Comment:
```suggestion
def verify_annotations_are_strings(
```
I think I would prefer this to be in the annotations module and import from there. Certainly would not look for it here, all the tests are in `test_annotations`, ... At a glance, I also don't see a function being used that is defined here, but I might have missed that.